Sources/Image+Tinting.swift

```diff
@@ -113,6 +116,11 @@ public extension BONImage {
             image = image.resizableImage(withCapInsets: originalCapInsets, resizingMode: originalResizingMode)
         }
 
+        // Transfer accessibility label (iOS only; watchOS does not have accessibilityLabel on UIImage).
+        #if os(iOS)
+        image.accessibilityLabel = self.accessibilityLabel
```

Comment:
nice-to-have: indent this line (project style: anything enclosed in `#if` should be indented.